internal/templates/00-assert.yaml.tmpl

```diff
@@ -4,6 +4,9 @@ timeout: {{ .TestCase.Timeout }}
 commands:
 - command: ${KUBECTL} annotate managed --all upjet.upbound.io/test=true --overwrite
 {{- range $resource := .Resources }}
+{{- if eq $resource.KindGroup "secret." -}}
```

Comment:
Why do we need a `.` in the kind group?

Reply:
It is because it group is empty (core group), and how it ends up with [this](https://github.com/upbound/uptest/blob/7cde0036c3d06f7ecd20742b33149ca33f699599/internal/tester.go#L63) kind-group calculation we are using.